vue/tailwind.config.cjs

```diff
@@ -6,5 +6,8 @@ module.exports = {
       mono: ["Iosevka", "monospace"],
     },
   },
-  plugins: [],
+  plugins: [require("daisyui")],
+  daisyui: {
+    themes: ['lofi'],
```

Comment:
I didn't put too much thought into this theme, but it seemed to match the existing theme of the app the best.